cg/cli/set/families.py

```diff
@@ -0,0 +1,77 @@
+import click
+from .family import family
+from cg.constants import FAMILY_ACTIONS, PRIORITY_OPTIONS
+from cg.store import models, Store
+
+CONFIRM = "Continue?"
+
+
+def _get_samples_by_identifiers(
+    identifiers: click.Tuple([str, str]), store: Store
+) -> [models.Sample]:
+    """Get samples matched by given set of identifiers"""
+    identifier_args = {}
+    for identifier_name, identifier_value in identifiers:
+        identifier_args[identifier_name] = identifier_value
+    return store.samples_by_ids(**identifier_args)
+
+
+def _get_cases(identifiers: click.Tuple([str, str]), store: Store) -> [models.Family]:
+    """Get cases that have samples that match identifiers if given"""
+    samples_by_id = _get_samples_by_identifiers(identifiers, store)
```

Comment:
```suggestion
    samples_by_id: List[models.Sample] = _get_samples_by_identifiers(identifiers, store)
```
I even start to to this, I think it is great

Reply:
I did know it was possible. If the called function has a proper return-type-hint isn't this overdoing it? I mean the IDE should know already..

Reply:
Sure It's more about readability but I guess that is a trade-off . Just wanted to tell that I started to use that but you do as you like 👍